src/role_scout.js

```diff
@@ -115,11 +115,11 @@ roles.scout.execute = function(creep) {
           delete creep.memory.last; // Delete to reset stuckness.
         }
       }
-      if (search.path.length > 0) {
-        creep.move(creep.pos.getDirectionTo(search.path[0]));
-      } else {
-        creep.moveTo(targetPosObject);
-      }
+      //if (search.path.length > 0) {
+      //creep.move(creep.pos.getDirectionTo(search.path[0]));
+      //} else {
+      creep.moveTo(targetPosObject);
+      //}
```

Comment:
Why do we want it to use the path is the pat his incomplete, the built path will be ov no use it it doesnt reach all the way?
 if (search.path.length > 0)

 seems of no use to us when path is inclomplete here 

if (search.incomplete || search.path.length === 0 || (creep.inBase() && creep.room.memory.misplacedSpawn)) {